pyproject.toml

```diff
@@ -42,8 +42,8 @@ inscriptis = "^1.1"
 # For some reason channels does not declare asigref and daphne as
 # Dependencies in the newest release of channels. Therefore we add
 # them here manually. Remove if this is fixed
-asgiref = "3.3.*"
-daphne = "2.*"
+asgiref = "3.*"
+daphne = "3.*"
```

Comment:
A major version bump in daphne? Really tested? Or is this purely because of the upgrade of channels to channels3?

Reply:
I checked the breaking changes introduced by these majors and they only talk about python versions supported changes (appart from some channels breaking changes that I already implemented)